web/www/missa/Latin/Commune/C6a.txt

```diff
@@ -40,11 +40,12 @@ Altíssimus.
 Veni, Sponsa Christi, áccipe corónam, quam tibi Dóminus præparávit in ætérnum: pro cujus amóre sánguinem tuum fudísti
 !Ps 44:8; 44:5
 Dilexísti justítiam, et odísti iniquitátem proptérea unxit te Deus, Deus tuus, óleo lætítiæ præ consórtibus tuis.
-V. Spécie tua et pulchritúdine tua inténde, próspere procéde et regna. Tempore autem Paschali omittitur
+V. Spécie tua et pulchritúdine tua inténde, próspere procéde et regna.
+! Tempore paschali omittitur graduale.
```

Comment:
This line can be omitted altogether, as "GradualeP" below will be used automatically during Paschaltide.